Added and updates some readmes

bettinaheim opened the pull request:
... such that I can directly link to the examples folder in the org readme. 
I fixed all formatting issues to comply with formatting rules, and the link check should also pass once I made the spec repo public.

idavis commented on examples/jit/README.md:
```diff
@@ -0,0 +1,26 @@
+# Examples: pyqir_jit
+
+The 'pyqir_jit' package provides an easy way to execute generated QIR for the
```
`pyqir_jit`

bettinaheim replied:
fixed here and where I copied it from

idavis reviewed (approved):
@bettinaheim Gate it a look through, only saw the one minor issue.

bettinaheim commented:
> I fixed all formatting issues to comply with formatting rules, and the link check should also pass once I made the spec repo public.

Thanks, @idavis! I fixed all formatting issues to comply with formatting rules, and the link check should also pass once I made the spec repo public.
